polkadot-protocol/onchain-governance/overview.md

```diff
@@ -1,4 +1,326 @@
+
+## Approval and Support
+
+The figure below summarizes the approval and support system during the **Decision Period**.
+
+![](/images/polkadot-protocol/onchan-governance/opengov-curves-pass.webp)
+
+Once a proposal exits the **Lead-in Period** and enters the **Voting Period**, it must meet both **`approval`** and **`support thresholds`** for the **Confirmation Period** to pass.
+
+**Approval:** The percentage of conviction-weighted **"aye"** votes against the total of conviction-weighted "aye" and "nay" votes.
+
+**Support:** The proportion of **"aye"** and **"abstain"** votes (unweighted by conviction) against the total possible votes (active issuance). In cases of split votes, only **"aye"** and **"abstain"** are counted.
+
+**Example:** Suppose total active issuance is 100 DOT:
+
+- Account A votes "Aye" with 10 DOT at 4x conviction
+
+- Account B votes "Nay" with 5 DOT at 2x conviction
+
+- Account C votes "Abstain" with 20 DOT (no conviction applies to abstain)
+
+With only 35 DOT participating, the values are calculated as follows:
+
+`Approval = Aye' / (Aye' + Nay’) = (10 × 4) / (10 × 4 + 5 × 2) = 40 / 50 = 80%`
+
+`Support = (Aye + Abstain) / (Total Active Issuance) = (10 + 20) / 100 = 30%`
+
+!!!note "Nay votes are not counted towards Support"
+    Support reflects voters who voted **"aye"** or chose to **"abstain"** excluding **"nay"** votes to avoid unintended outcomes. 
+    
+    This setup ensures that a **"nay"** vote cannot inadvertently push a referendum into the confirming state by raising support above the threshold without lowering approval.
+
+The figure illustrates the following points:
+
+1. A proposal enters the confirmation period only when both approval and support thresholds are met. Even if approval is above the approval curve, the proposal will wait until support also surpasses the support curve.
+
+2. If a referendum holds the approval and support thresholds for the entire confirmation period, it’s approved and scheduled for enactment. Tracks like Root enforce longer Enactment Periods to allow time for network adjustments, although proposers can opt for a longer enactment period.
+
+3. If the referendum exits the confirmation period (e.g., due to new "nay" votes), the confirmation clock resets. For instance, if it exits after 5 minutes in a 20-minute confirmation period, it must complete a fresh 20-minute confirmation upon re-entry.
+
+4. Should a referendum meet thresholds at the very end of the decision period, it can still pass if it confirms through the entire track-specific confirmation period. Otherwise, it’s rejected if it exits confirmation after the decision period ends.
+
+5. The approval curve starts at 100%, decreasing to a minimum of 50%. For a proposal to pass, weighted support must stay above 50%, assuming all tokens are actively voted.
```

Comment:
```suggestion
1. A proposal enters the confirmation period only when both approval and support thresholds are met. Even if approval is above the approval curve, the proposal will wait until support also surpasses the support curve

2. If a referendum holds the approval and support thresholds for the entire confirmation period, it’s approved and scheduled for enactment. Tracks like Root enforce longer Enactment Periods to allow time for network adjustments, although proposers can opt for a longer enactment period

3. The confirmation clock resets if the referendum exits the confirmation period (e.g., due to new "nay" votes). For instance, if it exits after 5 minutes in a 20-minute confirmation period, it must complete a fresh 20-minute confirmation upon re-entry

4. Should a referendum meet thresholds at the end of the decision period, it can still pass if confirmed through the entire track-specific confirmation period. Otherwise, it’s rejected if it exits confirmation after the decision period ends

5. The approval curve starts at 100%, decreasing to a minimum of 50%. For a proposal to pass, weighted support must stay above 50%, assuming all tokens are actively voted
```